tests/integration-tests/tests/configure/test_pcluster_configure.py

```diff
@@ -74,9 +131,26 @@ def get_unsupported_test_runner_options(request):
 
 
 def orchestrate_pcluster_configure(
-    region, key_name, scheduler, os, instance, vpc_id, public_subnet_id, private_subnet_id, config_path
+    region,
+    key_name,
+    scheduler,
+    os,
+    instance,
+    vpc_id,
+    public_subnet_id,
+    private_subnet_id,
+    config_path,
+    vpc_stack,
+    omitted_subnets_num=0,
 ):
     compute_units = "vcpus" if scheduler == "awsbatch" else "instances"
+    # Due to the order of creation,PublicSubnet is the first in the list of subnets. Therefore it is the default value.
+    vpc_stack_public_subnet_id = vpc_stack.cfn_outputs["PublicSubnetId"]
+    if omitted_subnets_num:
+        # When there are omitted subnets, a note should be printed
+        omitted_note = "Note:  {0} subnet\\(s\\) is/are not listed.+".format(omitted_subnets_num)
+    else:
+        omitted_note = ""
```

Comment:
could simplify the expression like 
`omitted_note = "Note:  {0} subnet\\(s\\) is/are not listed.+".format(omitted_subnets_num) if omitted_subnets_num else ""`